wandb/sdk/wandb_setup.py

```diff
@@ -128,7 +132,7 @@ def _early_logger_flush(self, new_logger):
         if not self._early_logger:
             return
         _set_logger(new_logger)
-        self._settings._clear_early_logger()
+        # self._settings._clear_early_logger()
```

Comment:
Comment
(if these are being left in because they're coming back, please feel free to ignore)

Reply:
yep, they are a note of old behavior that I need to think about